src/core/database.py

```diff
@@ -264,10 +281,52 @@ def _fieldsPlateform(self) -> QgsFields:
         fields = QgsFields()
         fields.append(self._createFieldShortText("ship"))
         fields.append(self._createFieldShortText("plateform"))
-        fields.append(self._createFieldShortText("plateformHeight"))
+        fields.append(QgsField("plateformHeight", QVariant.Double))
 
         return fields
 
+    def _populatePlateformTable(self) -> None:
+        plateformLyr = QgsVectorLayer(
+            self.tableUri(PLATEFORM_TABLE), "plateform", "ogr"
+        )
+        plateformLyr.startEditing()
+        plateforms = [
+            {"ship": "Thalassa", "plateform": "deck", "plateformHeight": 8.0},
+            {
+                "ship": "Thalassa",
+                "plateform": "bridge",
+                "plateformHeight": 14.0,
+            },
+            {
+                "ship": "Thalassa",
+                "plateform": "upper_bridge",
+                "plateformHeight": 16.5,
+            },
+            {"ship": "Europe", "plateform": "deck", "plateformHeight": 4.0},
+            {"ship": "Europe", "plateform": "bridge", "plateformHeight": 4.0},
+            {
+                "ship": "PourquoiPas",
+                "plateform": "deck",
+                "plateformHeight": 16.3,
+            },
+            {
+                "ship": "PourquoiPas",
+                "plateform": "bridge",
+                "plateformHeight": 18.6,
+            },
+            {
+                "ship": "PourquoiPas",
+                "plateform": "upper_bridge",
+                "plateformHeight": 21.6,
+            },
+        ]
```

Comment:
What do you think about adding a CSV file with these information? This way, it will be easier to update in the future.

Reply:
It's done, but above all it's to NOT have to update it in the future. User will update the csv with their values :)